app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,13 @@
+<footer class="footer">
+  <small>
+     <%= link_to t("home.message"), "https://www.railstutorial.org/" %>
+     <%= link_to t("by Michael Hartl"), "https://www.michaelhartl.com/" %>
```

Comment:
i18n key need define is by_michael_harrtl not by Michael Hartl
you cannot run t("by Michael Hartl"), it will translation missing